debug/src/codegen/v1/client.ts

```diff
@@ -1,5 +1,4 @@
-/** Do not modify. This file has been generated by @workleap/create-schemas */
+/** This file has been generated by @workleap/create-schemas (https://github.com/gsoft-inc/wl-openapi-typescript). Do not modify manually. */
```

Comment:
Unrelated to the current change, but this file had not been generated since the last PR, so this comment has only been updated now.